app/helpers/application_helper.rb

```diff
@@ -173,7 +173,7 @@ def safe_logo_url(logo)
   end
 
   def community_name
-    @community_name ||= SiteConfig.community_name # rubocop:disable Rails/HelperInstanceVariable
+    @community_name ||= SiteConfig.community_name
```

Comment:
Did this get smarter about setting instance variables in helpers vs accessing them so we could remove the cop?

Reply:
Apparently it's now aware of memoization: https://github.com/rubocop-hq/rubocop-rails/issues/311